deployment/wasm_pod_using_wde.yaml

```diff
@@ -0,0 +1,28 @@
+# Modified from krustlet hello world demo: https://github.com/deislabs/krustlet/blob/main/demos/wasi/hello-world-c/k8s.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: im-using-debug-echo-devices
+spec:
+  containers:
+    - name: im-using-debug-echo-devices
+      image: ghcr.io/rodz/wasi-hello-world:v1
```

Comment:
is this publically accessible or are build steps needed? Same for the other deployment yamls.

Reply:
Yes, all images are public \o/